images/harbor-database/Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM bitnami/postgresql:9.6
```

Comment:
do we really need the bitnami postgres here? couldn't we use the already existing postgres https://github.com/amazeeio/lagoon/blob/master/images/postgres/Dockerfile?

Reply:
Updated to using amazeeio's postgres image.